ui/src/main/resources/Meeting/SkinExtension.xml

```diff
@@ -133,6 +133,9 @@
 
 #durationEdit select{
    width:auto ;
+}
+.col-sm-12.col-xs-12 label {
+  padding-top: 12px;
```

Comment:
This is not the solution. We shouldn't add any explicit styling, but just rely on the Bootstrap grid. 
Maybe the classes and the structure is not correct. 

Also there is no difference between after.png and before.png.